aws/resource_aws_dx_hosted_private_virtual_interface.go

```diff
@@ -77,6 +77,16 @@ func resourceAwsDxHostedPrivateVirtualInterface() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validateAwsAccountId,
 			},
+			"mtu": {
+				Type:     schema.TypeInt,
+				Default:  1500,
+				Optional: true,
+				ForceNew: true,
```

Comment:
It looks like the MTU value for a VIF can be update via the [`UpdateVirtualInterface`](https://docs.aws.amazon.com/sdk-for-go/api/service/directconnect/#DirectConnect.UpdateVirtualInterfaceAttributes) API method, so `ForceNew` should not be set.
This will entail adding an `Update` method to the resources.